src/spec/color-spec/base-color-set.ts

```diff
@@ -58,4 +58,7 @@ export const BaseColorSet: ColorSet = {
             Background: "#FDF7F7", // Validation Background
         },
     } as ValidationTypes,
+    Shadow: {
+        Accent: "#57A9FF80",
```

Comment:
just a heads up that there will be a merge conflict for these color set files after #87 goes in